scripts/webpack/server.js

```diff
@@ -36,7 +35,10 @@ module.exports = () => ({
         rules: [
             {
                 test: /(\.tsx|\.js|\.ts)$/,
-                exclude: /node_modules\/(?!(@guardian\/discussion-rendering)|(@guardian\/types)|(dynamic-import-polyfill))\/.*/,
+                exclude: {
+                    and: [/node_modules/],
+                    not: [/@guardian/, /dynamic-import-polyfill/],
+                },
```

Comment:
Please check equivalence (though I did change it - we are looking at all `@guardian` modules on the server

Reply:
Webpack says:

> Try to avoid exclude and prefer include

Can we reverse this ^^?

Reply:
That said, i exclude makes sense for node_modules